packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,113 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=(tuxguitar tuxguitar-common tuxguitar-gtk2)
+pkgver=1.5.5
+pkgrel=2
+pkgdesc="Multitrack guitar tablature editor and player"
+arch=('x86_64' 'i686' 'armv7h')
+url="https://sourceforge.net/projects/tuxguitar/"
+license=('LGPL')
+depends=('alsa-lib' 'libxtst' 'glibc' 'java-runtime' 'hicolor-icon-theme')
+makedepends=('unzip' 'zip' 'ant' 'jack' 'fluidsynth' 'jdk-openjdk' 'maven')
+optdepends=('fluidsynth')
+source=(https://downloads.sourceforge.net/tuxguitar/tuxguitar-$pkgver-src.tar.gz
+        tuxguitar
+        tuxguitar-gtk2)
+sha256sums=('2e855500d9cea6987962a851d7e3fcb144e3220b10f71fba1514913e70542398'
+            'efeef39d43ecf5a87ed64abc7d8cf63a01f3c9b08bac0ea299bf959fcb7c216a'
+            '39f92c0de6fcf86635dec5ac3b83613ca980fa7d24f66888fd06e5bb2c7c571f')
+
+case $CARCH in
+  i686) _arch=x86;;
+  armv7h) _arch=armv7hl;;
+  *) _arch=$CARCH;;
+esac
+
+prepare() {
+  cd tuxguitar-$pkgver-src
+  export MAVEN_OPTS="$MAVEN_OPTS -Duser.home=$srcdir"
+  export JAVA_HOME=/usr/lib/jvm/default
+
+  for file in pom.xml TuxGuitar*/pom.xml build-scripts/*/pom.xml build-scripts/native-modules/*/pom.xml; do
+      sed -i -e 's|<source>1\.6</source>|<source>1\.7</source>|g' \
+             -e 's|<target>1\.6</target>|<target>1\.7</target>|g' "$file"
+
+      # https://maven.apache.org/guides/mini/guide-reproducible-builds.html
```

Comment:
I opened an upstream feature request for this already: https://sourceforge.net/p/tuxguitar/feature-requests/96/